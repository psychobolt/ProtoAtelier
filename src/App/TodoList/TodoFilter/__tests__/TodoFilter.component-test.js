import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../TodoFilter.component';

test('TodoFilter should render correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
