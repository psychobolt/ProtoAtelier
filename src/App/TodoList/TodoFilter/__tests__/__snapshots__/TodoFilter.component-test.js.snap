// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoFilter should render correctly 1`] = `
<x-tabs
  centered={null}
  style={Object {}}
>
  <withRouter(Connect(TodoFilterLink))
    filter="All"
  >
    All
  </withRouter(Connect(TodoFilterLink))>
  <withRouter(Connect(TodoFilterLink))
    filter="active"
  >
    Active
  </withRouter(Connect(TodoFilterLink))>
  <withRouter(Connect(TodoFilterLink))
    filter="completed"
  >
    Completed
  </withRouter(Connect(TodoFilterLink))>
</x-tabs>
`;
